schema/codelists/relatedProcess.csv

```diff
@@ -1,9 +1,10 @@
 Code,Title,Description
-framework,Framework agreement procedure first stage,This contracting process follows on from the related process to establish a framework. The suppliers on the framework are listed in the award section of the related process.
+framework,Framework agreement procedure first stage,This contracting process follows on from the related process that established a framework agreement. The suppliers on the framework are listed in the award section of the related process.
 planning,Planning process,This contracting process follows on from the related planning process.
 parent,Parent contract (for sub-contracts),This contracting process might result in a sub-contract of the related process.
-prior,Prior process,This contracting process is the renewal or replacement of the related prior process.
-unsuccessfulProcess,Unsuccessful process,This contracting process follows on from an previous unsuccessful process.
-subContract,Sub-contract,The related process might result in a sub-contract of this contract.
-replacementProcess,Replacement process,The related process might result in the replacement of this contract.
-renewalProcess,Renewal process,The related process might result in the renewal of this contract.
+prior,Prior process,This contracting process follows on from the related successful process.
+prior.replacement,Prior process (replacement),This contracting process is the replacement of the related process.
+prior.renewal,Prior process (renewal),This contracting process is the renewal of the related process.
+prior.additional,Prior process (additional),"This contracting process is for the purchase of additional goods, services or works related to the goods, services or works purchased from the same supplier in the related process."
+prior.repetition,Prior process (repetition),"This contracting process is for the purchase of new goods, services or works that constitute a repetition of the goods, services or works purchased from the same supplier in the related process."
+unsuccessfulProcess,Unsuccessful process,This contracting process follows on from the related unsuccessful process.
```

Comment:
```suggestion
unsuccessfulProcess,Unsuccessful process,The contracting process is another attempt to implement some contract(s). The related process is the contracting process that attempted and failed to implement the contract(s) previously.
```

Reply:
Same comment as https://github.com/open-contracting/standard/pull/1645/files#r1369449903

Reply:
I agree with changing "implement" to "conclude"

Reply:
Done in 1be3af934fe6ca688f387aa3777497f96220bae1